pages/recipe/index.js

```diff
@@ -0,0 +1,70 @@
+import Search from "@/component/Search";
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import Recipe from "@/component/Recipe";
+
+const app_id = "7c34a514";
+const app_key = "354397600a1db4ed480401dd1a84bc1e";
+
+export default function RecipePage() {
+  const [rescipes, setRescipes] = useState([]);
+  const [query, setQuery] = useState("");
+
+  useEffect(() => {
+    const fetchRecipes = async () => {
+      const URL = `https://api.edamam.com/api/recipes/v2?type=public&q=${query}&app_id=${app_id}&app_key=${app_key}&ingr=1-5&diet=balanced&calories=100-300`;
+      try {
+        const response = await fetch(URL);
+
+        if (!response.ok) {
+          throw new Error(
+            `Network response was not ok: ${response.statusText}`
+          );
+        }
+
+        const data = await response.json();
+        setRescipes(data.hits);
+      } catch (error) {
+        console.error("Fetch error:", error);
+      }
+    };
+
+    if (query) {
+      fetchRecipes();
+    } else {
+      setRescipes([]); // Clear the recipes list when the query is empty
+    }
+  }, [query]);
+
+  return (
+    <>
+      <Search query={query} setQuery={setQuery} />
+      <StyledHeading>Here are some ideas </StyledHeading>
```

Comment:
I think the ideas should be rendered at the first time you enter the page like as example ideas. But that is just happening when I write something in the input field and after I deleted the input the example ideas are shown. Correct me if I'm wrong :D so maybe the useEffect trigger is the problem.

Reply:
Hi, thank you for your feedback. The example ideas is intended to be empty(no example ideas shown)  if the query/search bar is empty :)

Reply:
Okay I understand first they are empty then I give an input and the examples are shown but when I delete my input there are still example ideas shown. So for me it would make sense that the ideas are just there if there is something in the input and if I delete the input there shouldn't be example ideas anymore. Just try it yourself, fill the input and then delete it again, then you will see what my point is.

Reply:
I tried it a few times and sometimes it is like you said and sometimes like I said, so I'm confused now :D

Reply:
Ahh it's because i had a space before my input word...sorry